y-octo/src/doc/codec/content.rs

```diff
@@ -219,7 +219,7 @@ impl Content {
         match self {
             Self::Deleted(len) => *len,
             Self::Json(strings) => strings.len() as u64,
-            Self::String(string) => string.encode_utf16().count() as u64,
+            Self::String(string) => string.chars().map(|c| c.len_utf16()).sum::<usize>() as u64,
```

Comment:
todo, we need to make custom wrapper with length cached, this cost too much

Reply:
in profiling, node.len() cost more time then string length calc, we can consider optimize somr first

Reply:
and if we cache the len, we can cache in higher layer like Node::Item to avoid the calc and atomic pointer jump

Reply:
that's other case, we still need a wrapper to support multiple char lengths of strings